pkg/controller/nginxingresscontroller/utils.go

```diff
@@ -143,6 +156,10 @@ func VerifySCCAPIExists() (bool, error) {
 
 	err = discovery.ServerSupportsVersion(clientSet, gv)
 	if err != nil {
+		// This error means the call could not find SCC in the API, but there was no API error.
+		if strings.Contains(fmt.Sprintf("%v", err), apiVersionUnsupportedError) {
```

Comment:
here it is also possible to use `err.Error()` to get the error message string. `fmt.Sprintf("%v", err)` uses `Error()` method to get the string value.